h/static/styles/partials-v2/_group-invite.scss

```diff
@@ -0,0 +1,63 @@
+.group-invite__title {
+  color: $brand;
+  margin-bottom: 0;
+}
+
+.group-invite__container {
+  position: relative;
+}
+
+.group-invite__input {
+  padding: 5px;
+  padding-top: 7px;
+  padding-bottom: 7px;
+  margin-top: 12px;
+  margin-bottom: 8px;
+  border: 1px solid $grey-3;
+  border-radius: 2px;
+  width: 100%;
+
+  .env-js-capable & {
+    padding-right: 22px;  /* Make space for the clipboard icon. */
+  }
+  .env-js-timeout & {
+    padding-right: 3px;
+  }
+}
+
+.group-invite__clipboard-button {
+  background: none;
+  height: 28px;
+  width: 23px;
+  border: none;
+  position: absolute;
+  right: 0;
+  top: 12px;
+
+  display: none;
+  .env-js-capable & {
+    display: block;
+  }
+  .env-js-timeout & {
+    display: none;
+  }
+}
+
+.group-invite__clipboard-button:focus {
+  outline: none;
+}
+
+.group-invite__clipboard-image {
+  // Vertically center the image inside its parent <button>.
+  margin-top: 5px;
+  margin-bottom: 5px;
+
+  width: 13px;
+  height: 16px;
+
+  fill: $grey-4;
+}
+
+.group-invite__clipboard-image:hover {
+  fill: $grey-7;
```

Comment:
This works in Chrome but not Firefox. It looks like the `fill` rule in the non-hovered state does work, but the `:hover` pseudo-class is not triggered.

I haven't investigated deeply, but this hack works around the problem by testing for the button rather than the `<svg>` element being hovered:

```diff
-.group-invite__clipboard-image:hover {
+button:hover > .group-invite__clipboard-image {
   fill: $grey-7;
 }
```